### PR TITLE
Set the word limit for the username input box

### LINKED ISSUE #607: Setting the word limit for the Username input box
Lengthy username is not supported and an error message is displayed rejecting the username updation.

### Steps to reproduce
Update the username and type a lengthy username. 

### Expected behaviour
To avoid the error message the word limit can be set. 

### Actual behaviour
The user is allowed to input the username of his choice but long username is not supported and the error message is displayed.

### Would you like to work on the issue?
Yes

### Screenshot
Username updation

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
## Description

Set the word limit to 50  for the username  input box.

Fixes #607 

## Type of change
Just put an x in the [] which are valid.
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [X] `./gradlew assembleDebug assembleRelease`
- [X] `./gradlew checkstyle`

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: Android/app/src/main/res/layout/activity_profile.xml [hunk 1]
```diff
@@ -140,6 +140,7 @@
                 android:ellipsize="end"
                 android:focusableInTouchMode="false"
                 android:inputType="textMultiLine"
+                android:maxLength="50"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_profile.xml]: is the UI going to be similar as status (I mean, the current count/total count will be shown by the side of edit text?)

Comment (ShivangiSingh17) [reply Android/app/src/main/res/layout/activity_profile.xml]: No, I will add that counter also.

Comment (ShivangiSingh17) [reply Android/app/src/main/res/layout/activity_profile.xml]: Made the suggested changes, added the word counter. @Swati4star can you please review now.

File: Android/app/src/main/res/layout/activity_profile.xml [hunk 2]
```diff
@@ -177,6 +178,18 @@
                 android:indeterminateDrawable="@drawable/progress_bar"
                 android:visibility="gone" />
 
+            <TextView
+                android:id="@+id/status_character_count"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignEnd="@+id/display_name"
+                android:layout_alignBottom="@+id/display_name"
+                android:text="@string/name_length"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_profile.xml]: where is this string `name_length`

this is causing build failure

**Comment by Swati4star** [comment]
fix the build & add screenshot

**Comment by ShivangiSingh17** [comment]
@Swati4star build error solved, can you please review the PR now.
PIC

**Pull request merged**